torch/csrc/jit/script/compiler.cpp

```diff
@@ -1424,6 +1427,31 @@ struct to_ir {
     }
   }
 
+  void emitAugAssignmentToSelectVar(const AugAssign& stmt) {
```

Comment:
Could you add a comment with an example of when this is called?

Reply:
added.